packages/gnu/gcc/gcc63.spk.yaml

```diff
@@ -1,4 +1,4 @@
-# {% set opt = opt | default_opts(version="6.3.3") %}
+# {% set opt = opt | default_opts(version="6.3.O") %}
```

Comment:
This is the letter `O` and not the digit `0` 🙃

Reply:
Oh, such a typo is embarrassing, my apologies for the lost time in fixing it.
This is a good reminder that even for quick fixes, local testing is not an option.

Thanks for the fix in https://github.com/spkenv/spk/commit/d9b67b7696fd14a6100c2c072905231c08b6c618 and https://github.com/spkenv/spk/commit/33ab16a1b53d5972f59b420e80e48b0747fb0406 @rydrman @jrray!

I have a question for the future: would it make sense to run the `Linux Build and Test` github action in PR's checks (like the one that failed after the merge: https://github.com/spkenv/spk/actions/runs/10288203486/job/28473074445) or would it [consume too many GitHub actions minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions) to be viable?

Reply:
> I have a question for the future: would it make sense to run the `Linux Build and Test` github action in PR's checks (like the one that failed after the merge: https://github.com/spkenv/spk/actions/runs/10288203486/job/28473074445) or would it [consume too many GitHub actions minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions) to be viable?

Yes, these checks are supposed to run for every PR but we have to manually approve running them for new contributors. As of today we've configured the project so CI needs to pass before a PR can merge.